Tasks/AzurePowerShell/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -23,5 +23,6 @@
   "loc.messages.InvalidScriptArguments0": "Invalid script arguments '{0}'. Line breaks are not allowed.",
   "loc.messages.InvalidScriptPath0": "Invalid script path '{0}'. Invalid path characters specified.",
   "loc.messages.InvalidAzurePsVersion": "The Azure PowerShell version '{0}' specified is not in the correct format. Please check the format. An example of correct format is 1.0.1",
-  "loc.messages.OverrideAzurePowerShellVersion": "The Azure PowerShell version '{0}' is not supported for Hosted Agent. Overriding it with version '{1}'. Consider changing the version selected."
+  "loc.messages.OverrideAzurePowerShellVersion": "The Azure PowerShell version '{0}' is not supported for Hosted Agent. Overriding it with version '{1}'. Consider changing the version selected.",
+  "loc.messages.CertificateEndpointNotSupported": "Service Principal enpoint with Certificate is not supported in 2.* task version. Upgrade the task version to 3.* to use the Certificate endpoint."
```

Comment:
this may not get translated, still we can try

Reply:
If this comes to master, then they will be part of localization process.